ldapadmin/src/main/webapp/console/app/components/user/user.es6

```diff
@@ -245,10 +245,8 @@ class UserController {
       'PENDING': false,
       'USER': true
     })
-    this.$injector.get('$cacheFactory').get('$http').removeAll()
-    this.$injector.get('$timeout')(() =>
-      this.groupPromise.$promise.then(this.bindGroups.bind(this))
-    )
+    let $httpDefaultCache = this.$injector.get('$cacheFactory').get('$http')
```

Comment:
We can keep orignal line : `this.$injector.get('$cacheFactory').get('$http').removeAll()`